### PR TITLE
remove the shadow under the instructions

### PR DESCRIPTION
**Pull request by chenlize96**
remove the shadow under the instructions

### CODE REVIEW

File: external/malmo/Minecraft/src/main/java/edu/arizona/tomcat/Mission/Goal/KillEntityGoal.java [hunk 1]
```diff
@@ -1,42 +1,65 @@
...
 			int playersY = (int) Minecraft.getMinecraft().player.posY;	
 			EntityZombie n = new EntityZombie(world);
-			n.setPosition(playersX + distance, playersY, playersZ);
-			//world.onEntityAdded(n); // draw an entity cannot move
+			n.setPosition(playersX, playersY, playersZ  + distance);
+			UUID id = UUID.randomUUID(); // generate new id for new entity
+			idList.add(id); // keep both idList and willBeKilled here, they have different functions
+			willBeKilled.add(id);
+			n.setUniqueId(id);		
+			//world.onEntityAdded(n); // draw an entity cannot move (save this line, it maybe used to apply AI)
```
Comment (adarshp) [inline external/malmo/Minecraft/src/main/java/edu/arizona/tomcat/Mission/Goal/KillEntityGoal.java]: Please delete this commented out line.

File: external/malmo/Minecraft/src/main/java/edu/arizona/tomcat/Mission/gui/GuiBackground.java [hunk 2]
```diff
@@ -84,14 +84,30 @@ public void drawLabel(Minecraft mc, int mouseX, int mouseY)
...
+
+	 /**
+	  * Renders the specified text to the screen. Args : renderer, string, x, y, color
+	  */
+	 @Override
+	 public void drawString(FontRenderer fontRendererIn, String text, int x, int y, int color)
+	 {
+		 fontRendererIn.drawString(text, (float)x, (float)y, color, false);
+	 }
+	 //@Override
```
Comment (adarshp) [inline external/malmo/Minecraft/src/main/java/edu/arizona/tomcat/Mission/gui/GuiBackground.java]: Please delete this commented out line.

File: external/malmo/Minecraft/src/main/java/edu/arizona/tomcat/Mission/Goal/KillEntityGoal.java [hunk 3]
```diff
@@ -1,42 +1,65 @@
...
+					this.goalAchieved = true;
 				}
 			}
-			if (sum == 1) {
-				this.goalAchieved = true;
-			}
 		}
-		
-		
+
```
Comment (adarshp) [inline external/malmo/Minecraft/src/main/java/edu/arizona/tomcat/Mission/Goal/KillEntityGoal.java]: Please delete the unnecessary blank lines.

File: external/malmo/Minecraft/src/main/java/edu/arizona/tomcat/Mission/Goal/KillEntityGoal.java (hunk 1 above)
Comment (chenlize96) [reply external/malmo/Minecraft/src/main/java/edu/arizona/tomcat/Mission/Goal/KillEntityGoal.java]: I am not sure why this class was modified. I only changed code in GuiBackground.java

**Pull request closed without merging**